tests/arm/Tests.lst

```diff
@@ -94724,3 +94724,67 @@ MaxAllowedDurationSeconds=600
 Categories=EXPECTED_PASS
 HostStyle=0
 
+[GitHub_18522.cmd_11900]
+RelativePath=JIT\Regression\JitBlue\GitHub_18522\GitHub_18522\GitHub_18522.cmd
+WorkingDir=JIT\Regression\JitBlue\GitHub_18522\GitHub_18522
+Expected=0
+MaxAllowedDurationSeconds=600
+Categories=EXPECTED_PASS
+HostStyle=0
+
+[GitHub_18522_1.cmd_11901]
+RelativePath=JIT\Regression\JitBlue\GitHub_18522\GitHub_18522_1\GitHub_18522_1.cmd
+WorkingDir=JIT\Regression\JitBlue\GitHub_18522\GitHub_18522_1
+Expected=0
+MaxAllowedDurationSeconds=600
+Categories=EXPECTED_PASS
+HostStyle=0
+
+[GitHub_18522_2.cmd_11902]
+RelativePath=JIT\Regression\JitBlue\GitHub_18522\GitHub_18522_2\GitHub_18522_2.cmd
+WorkingDir=JIT\Regression\JitBlue\GitHub_18522\GitHub_18522_2
+Expected=0
+MaxAllowedDurationSeconds=600
+Categories=EXPECTED_PASS
+HostStyle=0
+
+[GitHub_18522_3.cmd_11903]
+RelativePath=JIT\Regression\JitBlue\GitHub_18522\GitHub_18522_3\GitHub_18522_3.cmd
+WorkingDir=JIT\Regression\JitBlue\GitHub_18522\GitHub_18522_3
+Expected=0
+MaxAllowedDurationSeconds=600
+Categories=EXPECTED_PASS
+HostStyle=0
+
+[GitHub_18522_4.cmd_11904]
+RelativePath=JIT\Regression\JitBlue\GitHub_18522\GitHub_18522_4\GitHub_18522_4.cmd
+WorkingDir=JIT\Regression\JitBlue\GitHub_18522\GitHub_18522_4
+Expected=0
+MaxAllowedDurationSeconds=600
+Categories=EXPECTED_PASS
+HostStyle=0
+
+[GitHub_18522_5.cmd_11905]
+RelativePath=JIT\Regression\JitBlue\GitHub_18522\GitHub_18522_5\GitHub_18522_5.cmd
+WorkingDir=JIT\Regression\JitBlue\GitHub_18522\GitHub_18522_5
+Expected=0
+MaxAllowedDurationSeconds=600
+Categories=EXPECTED_PASS
+HostStyle=0
+
+[GitHub_18522_6.cmd_11903]
```

Comment:
`_11906`? Same below.

Reply:
Is this actually important?

Reply:
Not sure -- but now I remember there is a tool to regenerate these, so I suuppose we should just use that.

tests\scripts\lst_creator.py

It would be nice if the tool output included the command line used to create that output.

Reply:
The tool is a little funky because it does a recursive discovery of the tests. Hopefully with the work @sbomer is doing this legacy stuff just goes away. I think the easier approach here is to just keep it as it is, the numbering is not extremely important, it is just output by the tool to make sure the tags are unique. GitHub_18522_6.cmd_11903 should be unique enough.

Reply:
Note that the tool will strip the number anyways, so this is not going to break its input either.